torch/testing/_internal/common_fsdp.py

```diff
@@ -32,22 +33,6 @@ class FSDPInitMode(Enum):
     # Don't move model to CUDA at all.
     CUDA_NEVER = 3
 
-# get full params of a model recursively. Note that if CPU offloading, it will
-# also automatically move the parameters to GPU, due to _rebuild_full_params
-# call.
-def get_full_params(model, recurse=True):
-    if recurse:
-        # get all params for any nested FSDP instances.
-        for module in model.modules():
-            if isinstance(module, FullyShardedDataParallel):
-                get_full_params(module, recurse=False)
-    else:
-        torch.cuda.synchronize()
-        model._rebuild_full_params()
-        torch.cuda.synchronize()
-        if model.module.flat_param is not None:
-            model.module._unflatten_params()
-
```

Comment:
just replace this with following? so that other places do not need to change?

            with model._summon_full_params():
                return deepcopy(list(model.parameters()))